### PR TITLE
chore(test): clean up no control flow typescript tests

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

File: spec/ts/basic/element_spec.ts [hunk 1]
```diff
@@ -1,186 +1,191 @@
...
+  it('should chain element actions', async() => {
     await browser.get('index.html#/form');
 
     const usernameInput = element(by.model('username'));
     const name = element(by.binding('username'));
 
-    await expect(name.getText()).toEqual('Anon');
+    expect(await name.getText()).toEqual('Anon');
 
     await((usernameInput.clear() as any) as ElementFinder).sendKeys('Jane');
```
[the remark this answers is not in the document]
Comment (cnishina) [reply spec/ts/basic/element_spec.ts]: Yes. wat is right. I might need to fix this.

[the remark this answers is not in the document]
Comment (cnishina) [reply spec/ts/basic/element_spec.ts]: Casting it just directly to `ElementFinder` does not work as shown above. Per the documentation, the return type is `webdriver.promise.Promise.<void>` so I'm not sure how this actually works. Shouldn't it return a promise for an ElementFinder? That would make more sense.

[the remark this answers is not in the document]
Comment (heathkit) [reply spec/ts/basic/element_spec.ts]: Oh right, yeah it should be 'as Promise<ElementFinder>'

[the remark this answers is not in the document]
Comment (cnishina) [reply spec/ts/basic/element_spec.ts]: I added a TODO to fix this and it is logged as an issue.

**Pull request merged**